godot-kotlin/godot-library/src/nativeMain/kotlin/godot/registration/RPCMode.kt

```diff
@@ -0,0 +1,14 @@
+package godot.registration
+
+//TODO: temporary enum class! Needs to be properly implemented! Implementation was done to be able to implement register annotations
+enum class RPCMode {
+    Disabled,
```

Comment:
We should be consistent on how we capitalize enum values. On my PR I had `Platform` enum values all caps. Should we do the same here? Or maybe the reverse?

Reply:
The stdlib way is having it all caps and underscore separated when needed.

Reply:
yeah here as well: https://kotlinlang.org/docs/reference/enum-classes.html
I'll change it to all caps :-)